README.md

```diff
@@ -144,6 +150,10 @@ jobs:
       # Run the commitlint action, considering its own dependencies and yours as well 🚀
       # `github.workspace` is the path to your repository.
       - uses: wagoid/commitlint-github-action@v5
+        # Optinally, add a commitDepth parameter
+        # This example would only consider the last 10 commits.
+        with:
```

Comment:
Could you remove this part? This section is intended to show how to include custom packages via `NODE_PATH`, I think it's better to keep focused on the `NODE_PATH` instructions so that people don't get confused

Reply:
Updated.